.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+set -eux
+
+TARGETOS=linux
+TARGETARCH=amd64
+
+# Install Kind
+curl -L -o kind "https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-${TARGETARCH}" && chmod +x ./kind
+sudo mv ./kind /usr/local/bin
+
+# Install kubectl
```

Comment:
Isn't `kubectl` installed by default on codespace? This is what I see when I forked `idpbuilder` repo and created a codespace .

```
@sunnygoel87 ➜ /workspaces/idpbuilder (main) $ kubectl version
Client Version: v1.29.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```